modules/fastq/templates/merge_cram.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+
+main() {
+    IFS=' ' read -a cram_array <<< "!{crams}";
+    if [ ${#cram_array} -gt 1 ]
+    then
+        ${CMD_SAMTOOLS} merge -@ "!{task.cpus}" -o !{cramOut} --write-index "${cram_array}"
```

Comment:
`merge` process should not be called if a merge is not required?

Reply:
This is needed to get the publishing of the file right. You also want to publish the cram if no merge was required.